Doppler.Push.Api/Controllers/DeviceTokenController.cs

```diff
@@ -0,0 +1,21 @@
+using Doppler.Push.Api.Contract;
+using Doppler.Push.Api.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace Doppler.Push.Api.Controllers
+{
+    [Authorize(Policies.ONLY_SUPERUSER)]
+    [ApiController]
+    public class DeviceTokenController : ControllerBase
+    {
+        [HttpGet]
+        [Route("device-tokens/{deviceToken}")]
+        public async Task<ActionResult<DeviceTokenInfo>> GetInfo([FromRoute] string deviceToken)
```

Comment:
Why do you have a resource to represent the _token of a device_, in place of having a resource to represent the _device_ himself?

```suggestion
        [HttpGet]
        [Route("devices/{deviceToken}")]
        public async Task<ActionResult<Device>> Get([FromRoute] string deviceToken)
```

Reply:
Great idea! Thanks

Reply:
@andresmoschini I am thinking on change `deviceToken` route param to `token`:
```suggestion
        [HttpGet]
        [Route("devices/{token}")]
        public async Task<ActionResult<Device>> Get([FromRoute] string token)
```
`Device` model will be:
```
public class Device
    {
        public string Token { get; set; }

        public bool IsValid { get; set; }
    }
```

Can you give me your opinion?

Reply:
Looks nice!